examples/next-js/next.config.js

```diff
@@ -0,0 +1,52 @@
+const RollbarSourcemapPlugin = require('rollbar-sourcemap-webpack-plugin');
+
+const nextConfig = {
+  webpack: (config, { dev, webpack }) => {
+    if (!dev) {
+      // generate sourcemaps
+      config.output.futureEmitAssets = false;
+      config.devtool = 'nosources-source-map';
+
+      for (const plugin of config.plugins) {
```

Comment:
Why didn't you just use [next-source-maps](https://github.com/zeit/next-plugins/tree/master/packages/next-source-maps)?

Reply:
Originally this was a combination of a few different projects that had custom needs so we weren't using `@zeit/next-source-maps` in those. I updated it to use that instead.